tests/test_validate_metadata.py

```diff
@@ -125,18 +139,57 @@ def test_invalid_nonontology_content(self):
         reference_file = open("../tests/data/metadata_invalid.json", "r")
         reference_issues = json.load(reference_file)
         reference_file.close()
-        serialize_issues(metadata)
-        current_file = open("issues.json", "r")
-        current_issues = json.load(current_file)
-        current_file.close()
         self.assertEqual(
-            current_issues,
+            metadata.issues,
             reference_issues,
             "Metadata validation issues do not match reference issues",
         )
 
         self.teardown_metadata(metadata)
 
+    def test_valid_ontology_content(self):
+        """Ontology metadata should conform to convention requirements
+            """
+        args = "../tests/data/AMC_v1.1.00.json " "../tests/data/ontology_valid.tsv"
```

Comment:
Simpler equivalent:

```suggestion
        args = "../tests/data/AMC_v1.1.00.json ../tests/data/ontology_valid.tsv"
```

(I had never seen that syntax before!)

Reply:
fixed multiple instances; see `4023a08`
not sure why I had that syntax...